src/seedu/addressbook/commands/employee/EmployeeDeleteCommand.java

```diff
@@ -2,19 +2,21 @@
 
 import seedu.addressbook.commands.Command;
 import seedu.addressbook.commands.CommandResult;
-import seedu.addressbook.common.Messages;
 import seedu.addressbook.data.employee.ReadOnlyEmployee;
 import seedu.addressbook.data.employee.UniqueEmployeeList.EmployeeNotFoundException;
+
+import seedu.addressbook.common.Messages;
+
 /**
- * Deletes an employee identified using it's last displayed index from the address book.
+ * Deletes an employee identified using it's last displayed index from the Rms.
  */
 public class EmployeeDeleteCommand extends Command{
 
     public static final String COMMAND_WORD = "delemp";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
             + "Deletes the employee identified by the index number used in the last employee listing.\n\t"
-            + "Parameters: INDEX\n\t"
+            + "Parameters: INDEX (must be a positive integer)\n\t"
```

Comment:
I don't think the note in the bracket is useful. Instead, you can write it in the user guide